spread.yaml

```diff
@@ -9,7 +9,10 @@ environment:
     PATH: $GOPATH/bin:/snap/bin:$PATH:/usr/lib/go-1.6/bin
     TESTSLIB: $PROJECT_PATH/tests/lib
     SNAPPY_TESTING: 1
-    SNAP_REEXEC: 0
+    # we run the entire suite with re-exec on (the default) and modify
+    # the core snap so that it contains our new code.  So we run new
+    # snapd from the deb that re-execs into new snapd in core.  To
+    # test purely from the deb, the "SNAP_REEXEC: 0" here
```

Comment:
We could add here an environment variable so that we can parameterize the modification of the core snap later, something like:
```
MODIFY_CORE_SNAP_FOR_REXEC: "$(HOST: echo ${SPREAD_MODIFY_CORE_SNAP_FOR_REXEC:-1})"
```

This way we could control the modification of the core snap from the host, modifying it by default.